css/store.css

```diff
@@ -0,0 +1,136 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    
+    margin: 0.5rem 1rem;
```

Comment:
margin:0.5 em 1em;
it s better to use em foe margin and padding

Reply:
Thank you for reviewing my cod.